opentech/storage_backends.py

```diff
@@ -24,4 +27,16 @@ class PrivateMediaStorage(S3Boto3Storage):
     encryption = True
     file_overwrite = False
     querystring_auth = True
-    url_protocol = 'https:'
+    secure_urls = True
```

Comment:
This change give the same result but it looks like url_protocol is the current version of this setting.

https://github.com/jschneier/django-storages/blob/master/storages/backends/s3boto3.py#L251

Reply:
OK, i guess i prefer the secure urls argument. But you are current, `url_protocol` is the correct one to use with boto3